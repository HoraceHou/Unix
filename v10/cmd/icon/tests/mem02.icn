procedure main()
   write(image(x := []) | "failed")
   write(image(every push(x,1 to 1000)) | "failed")
   write(image(x) | "failed")
   write(image(every push(x,1 to 10000)) | "failed")
   write(image(x) | "failed")
end
