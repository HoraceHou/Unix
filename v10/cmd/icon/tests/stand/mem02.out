list(0)
failed
list(1000)
failed
list(11000)
